maths/mobius_function.py

```diff
@@ -0,0 +1,42 @@
+"""
+Refrences: https://en.wikipedia.org/wiki/M%C3%B6bius_function
+References: wikipedia:square free number
+python/black : True
+flake8 : True
+"""
+
+from typing import List

```

Comment:
./maths/mobius_function.py:8:1: F401 'typing.List' imported but unused

Reply:
I fixed it. Flake 8 passed.